malwareCheck.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
+import subprocess
+import glob
+import os
+
+# Check for malware, as per https://support.apple.com/en-us/ht203987
```

Comment:
please verify this is up to date, although we should really be using SavingThrow